packages/inferno/src/DOM/utils/common.ts

```diff
@@ -9,12 +9,12 @@ export const EMPTY_OBJ = {};
 export const Fragment: string = '$F';
 
 export class AnimationQueues {
```

Comment:
I have been thinking about this animations / lifecycle parameter that maybe those should be combined into "renderContext" or something

Reply:
Great idea! I can look into this once I have documentation done.